app/styles/main.scss

```diff
@@ -323,6 +323,18 @@ form {
   }
 }
 
+reload div {
+  text-align: center;
+  padding: 10px 30px;
+  background-color: lighten($brand-primary-light, 15%);
```

Comment:
i can't decide between this color and `$gray-light`. can you try both and let me know what you prefer?